test_ert_integration_errors: FileNotFoundError `ert-log.txt`

Bleeding have a failing test regarding the `ert-log.txt` file which is not found
```
norne_mocked_ensembleset = None
snapshot = <pytest_snapshot.plugin.Snapshot object at 0x7f5e9e796f40>

    @pytest.mark.ert_integration
    def test_ert_integration_errors(norne_mocked_ensembleset, snapshot):
        """Test CSV_EXPORT2 when runpathfile points to non-existing realizations
    
        This test proves that CSV_EXPORT2 happily skips non-existing
        realizations, but emits a warning that there is no STATUS file.
        """
        with open("FOO.DATA", "w", encoding="utf-8") as file_h:
            file_h.write("--Empty")
    
        # Append a not-existing realizations to the runpathfile:
        with open("runpathfile", "a", encoding="utf-8") as file_h:
            file_h.write("002 realization-2/iter-0 NORNE_1 000")
    
        with open("wf_csvexport", "w", encoding="utf-8") as file_h:
            file_h.write("CSV_EXPORT2 runpathfile data.csv monthly FOPT\n")
    
        ert_config = [
            "ECLBASE FOO.DATA",
            "QUEUE_SYSTEM LOCAL",
            "NUM_REALIZATIONS 2",
            "LOAD_WORKFLOW wf_csvexport",
            "HOOK_WORKFLOW wf_csvexport PRE_SIMULATION",
        ]
    
        ert_config_fname = "test.ert"
        with open(ert_config_fname, "w", encoding="utf-8") as file_h:
            file_h.write("\n".join(ert_config))
    
        subprocess.run(["ert", "test_run", ert_config_fname], check=True)
    
>       with open("ert-log.txt") as fin:
E       FileNotFoundError: [Errno 2] No such file or directory: 'ert-log.txt'
```

--- a/tests/jobs/csv_export2/test_integration.py
+++ b/tests/jobs/csv_export2/test_integration.py
@@ -260,8 +260,12 @@ def test_ert_integration_errors(norne_mocked_ensembleset, snapshot):
         file_h.write("\n".join(ert_config))
 
     subprocess.run(["ert", "test_run", ert_config_fname], check=True)
-
-    with open("ert-log.txt") as fin:
+    log_file = "ert-log"
+    for file in os.listdir():
+        if file.startswith(log_file):
+            log_file = file
+            break
+    with open(log_file) as fin:
         ertlog = fin.read()
     assert "fmu.ensemble.realization - WARNING - No STATUS file" in ertlog
     assert "realization-2/iter-0" in ertlog

